src/main/scala/org/globalforestwatch/summarystats/afi/AFiAnalysis.scala

```diff
@@ -42,17 +41,36 @@ object AFiAnalysis extends SummaryAnalysis {
 //        }
 //    }.unify
 
-
     // TODO somehow convert AFiSummary to AFiData
     import spark.implicits._
 
-    val summaryDF = AFiDF.getFeatureDataFrame(summaryRDD, spark)
+    val summaryDF = AFiDF
+      .getFeatureDataFrame(summaryRDD, spark)
+      .withColumn("gadm_id", when(col("location_id") =!= -1, lit("")).otherwise(col("gadm_id")))
+
+    val gadmAgg = summaryDF
```

Comment:
maybe I'm reading, but I think you may have gotten it reversed - for GADM we want to group by GADM id